lib/Agrammon/Model.pm6

```diff
@@ -156,30 +157,37 @@ class Agrammon::Model {
         method !annotate-inputs-internal($input-data, %run-already, @result, Str $instance-id?) {
             my $tax = $!module.taxonomy;
             return if %run-already{$tax};
+            my $gui-root-module = $!module.gui-root-module;
+            my %gui-root;
+            if $gui-root-module {
+                # what a silly inline signaling
+                my @g = $gui-root-module.gui.split(',');
+                %gui-root = %( :de(@g[1]), :fr(@g[2]), :en(@g[3] // @g[0])) ;
```

Comment:
Since you're assigning into a hash, you don't need the `%( ... )` on the right; a list of pairs will do fine.

Reply:
Ah, I forgot. Just fixed.